client/src/components/hooks/useServiceWorker.js

```diff
@@ -162,5 +166,6 @@ export {
 	parseMessageData,
 	rejectSignMessageRequest,
 	rejectSignDeployRequest,
-	cancelConnectingSite
+	cancelConnectingSite,
+	isPopupWindow
```

Comment:
@hoailinh1210 Please help to change the popup name e.g isSigningPopupWindow, it will help to differentiate with browser extension popup

Reply:
Hi @djhahe, I updated it